### ISSUE TITLE
Installation large IPA file error

### ISSUE THREAD

**Issue by srdr0p** [doronz88/pymobiledevice3#395]
**Test environment**
- version: 1.36.0
- MacOS 12.3
- iPhone 6 12.3/ iPhone SE 2 14.0

**Describe the bug**
Installation fail when ipa is too large.

**To Reproduce**
1. Use InstallationProxyService to install large IPA (about 2.4GB)

**Expected behavior**
Install successful.

**Logs**
```
  File "/Users/zephyrxiao/Documents/iOS/IPACrawler/ipa/test.py", line 6, in <module>
    instll_svc.install("./com.tencent.jkchess_1.4.11 (888).ipa")
  File "/opt/homebrew/lib/python3.10/site-packages/pymobiledevice3/services/installation_proxy.py", line 66, in install
    return self.install_from_local(ipa_path, 'Install', options, handler, args)
  File "/opt/homebrew/lib/python3.10/site-packages/pymobiledevice3/services/installation_proxy.py", line 56, in install_from_local
    afc.set_file_contents(remote_path, open(ipa_path, 'rb').read())
  File "/opt/homebrew/lib/python3.10/site-packages/pymobiledevice3/services/afc.py", line 496, in set_file_contents
    self.fwrite(h, data)
  File "/opt/homebrew/lib/python3.10/site-packages/pymobiledevice3/services/afc.py", line 459, in fwrite
    self._dispatch_packet(afc_opcode_t.WRITE,
  File "/opt/homebrew/lib/python3.10/site-packages/pymobiledevice3/services/afc.py", line 542, in _dispatch_packet
    self.service.sendall(header + data)
  File "/opt/homebrew/lib/python3.10/site-packages/pymobiledevice3/service_connection.py", line 129, in sendall
    self.socket.sendall(data)
OSError: [Errno 22] Invalid argument
```
**Additional context**
I try to modify `MAXIMUM_WRITE_SIZE` to `1 << 30` in afc.py, and the installation is successful. 
[taobao-iphone-device](https://github.com/alibaba/taobao-iphone-device/blob/e5976e9390f6330329c105c0f0a7ed0818f3836c/tidevice/_sync.py#L392)  use chunksize 1 << 15，and also no problem.

**Comment by doronz88** [comment]
@srdr0p Thanks for reporting (and the suggested fix)! 🙂
Can you verify #396 solves the issue? If so, I'll create this as an official release

**Comment by srdr0p** [comment]
@doronz88 Yes, it solves the issue.

**Comment by doronz88** [comment]
@srdr0p I just released [v1.36.3](https://github.com/doronz88/pymobiledevice3/releases/tag/v1.36.3) with this bugfix. You are welcome to update 🥳
